src/components/global/DataGraph.vue

```diff
@@ -0,0 +1,209 @@
+                } else if (null != detail.provinceState && null == detail.countryRegion) {
+                    return detail.provinceState;
+                } else {
+                    return 'n/a';
+                }
+            },
+            formatNumber(x) {
+                return numeral(x).format('0,0');
+            },
+            changeCategory(activeTab) {
+                this.activeTab = activeTab
+                let categories = []
+                let dataSeries = []
+                let nameSeries = 'Sembuh'
+                let chartHeight = 1000
+                let total = 0
+
+                switch (this.activeTab) {
+                    case 'positive':
+                        total = this.casePositive.reduce(function (acc, obj) { return acc + obj.kasusPosi; }, 0);
+                        nameSeries = 'Positif'
+                        categories = this.casePositive.map(item => { return item.provinsi })
+                        dataSeries = this.casePositive.map(item => {
+                            return {name: item.provinsi, y: item.kasusPosi, percentage: (item.kasusPosi/total * 100).toFixed(2)}
+                        })
+                        break;
+                    case 'died':
+                        total = this.casePositive.reduce(function (acc, obj) { return acc + obj.kasusMeni; }, 0);
+                        nameSeries = 'Meningggal'
+                        categories = this.caseDied.map(item => { return item.provinsi })
+                        dataSeries = this.caseDied.map(item => {
+                            return {name: item.provinsi, y: item.kasusMeni, percentage: (item.kasusMeni/total * 100).toFixed(2)}
+                        })
+                        break;
+                    default:
+                        total = this.casePositive.reduce(function (acc, obj) { return acc + obj.kasusSemb; }, 0);
+                        categories = this.caseRecovery.map(item => { return item.provinsi })
+                        dataSeries = this.caseRecovery.map(item => {
+                            return {name: item.provinsi, y: item.kasusSemb, percentage: (item.kasusSemb/total * 100).toFixed(2)}
+                        })
+                        break;
+                }
+
+                if (this.showTop10) {
+                    chartHeight = 500
+                    categories = categories.slice(0, 10)
+                    dataSeries = dataSeries.slice(0, 10)
+                }
+
+                this.chartOption.chart.height = chartHeight
+                this.chartOption.xAxis.categories = categories
+                this.chartOption.series = [{name: nameSeries, data: dataSeries}]
+            },
+            showAllProvince() {
+                this.showTop10 = !this.showTop10
+                this.changeCategory(this.activeTab)
+
+                // Scroll to the top of chart
+                const element = document.getElementById('chart-indonesia')
```

Comment:
please check this out, make sure to scroll up to the global section, not to Indonesia

Reply:
sorry @pararang .. I forget not to review it again, i'll fix it later